### PR TITLE
Added Commands API

### PR DESCRIPTION
**Pull request by kk25q**
Ready For Review.

### CODE REVIEW

**Comment by kristinpeterson** [comment]
@sarahhenkens when you have a moment, this is ready for review.

File: examples/device_commands.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   catch (M2XException $ex) {
+     echo 'Error: ' . $ex->getMessage();
+     echo $ex->response->raw;
+     break;
+   }
+}
+
+function listCommands() {
+  try {
+
```
Comment (sarahhenkens) [inline examples/device_commands.php]: Unnecessary empty line.

File: src/Command.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+/**
+ * The resource id for the REST URL
+ *
+ * @return string
+ */
+  public function id() {
+    return $this->id;
+  }
+
+  public function refresh() {
```
Comment (sarahhenkens) [inline src/Command.php]: Missing docblock.

File: src/Device.php [hunk 3]
```diff
@@ -244,4 +244,56 @@ public function log() {
...
+     return $this->client->post($this->path() . $command->path() . '/reject', $data);
+  }
+
+/**
+ * Device marks Command as processed
+ *
+ * @link https://m2x.att.com/developer/documentation/v2/commands#Device-Marks-a-Command-as-Processed
+ *
+ * @param array $data
+ * @param Command $command
```
Comment (sarahhenkens) [inline src/Device.php]: These 2 params should flip their position. `$command` is first in the parameter list.

**Comment by kk25q** [comment]
Requested Changes are Completed.Ready for Review.

**Pull request merged**